### PR TITLE
docs: Update docs for new StratoVirt VMM introduction

### PR DESCRIPTION
**Pull request by fadecoder**
As the StratoVirt VMM has been added, we can update the docs and make some intoduction to StratoVirt, thus users can know more about the hypervisor choices.

Fixes: #8645

### CODE REVIEW

**Comment by WenyuanLau** [comment]
Thanks! @fadecoder 

Can anyone in Documentation Team take a look?
cc @kata-containers/documentation

File: tools/packaging/kata-deploy/README.md [hunk 1]
```diff
@@ -254,7 +270,7 @@ applying labels to the nodes.
 This DaemonSet installs the necessary Kata binaries, configuration files, and virtual machine artifacts on
 the node. Once installed, the DaemonSet adds a node label `katacontainers.io/kata-runtime=true` and reconfigures
 either CRI-O or containerd to register three `runtimeClasses`: `kata-clh` (for Cloud Hypervisor isolation), `kata-qemu` (for QEMU isolation),
-and `kata-fc` (for Firecracker isolation). As a final step the DaemonSet restarts either CRI-O or containerd. Upon deletion,
+, `kata-fc` (for Firecracker isolation) and `kata-stratovirt` (for StratoVirt isolation). As a final step the DaemonSet restarts either CRI-O or containerd. Upon deletion,
```
Comment (stevenhorsman) [inline tools/packaging/kata-deploy/README.md:273]: nits: previous line already ends in a comma, Add a new line break to keep doc lines to the recommended 120 chars

```suggestion
`kata-fc` (for Firecracker isolation) and `kata-stratovirt` (for StratoVirt isolation).
As a final step the DaemonSet restarts either CRI-O or containerd. Upon deletion,
```

Comment (fadecoder) [reply tools/packaging/kata-deploy/README.md]: thanks，updated.

**Review by stevenhorsman — APPROVED** [review]
Docs looks okay to me, running the tests now to check there aren't any issues, but I can't envisage any

**Comment by WenyuanLau** [comment]
> Docs looks okay to me, running the tests now to check there aren't any issues, but I can't envisage any

Thanks! @stevenhorsman

**Review by WenyuanLau — APPROVED** [review]

**Review by GabyCT — APPROVED** [review]

File: docs/design/virtualization.md [hunk 2]
```diff
@@ -112,10 +112,27 @@ Devices and features used:
 - seccomp filters
 - [HTTP OpenAPI](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/main/vmm/src/api/openapi/cloud-hypervisor.yaml)
 
+### StratoVirt/KVM
+
+[StratoVirt](https://gitee.com/openeuler/stratovirt) is an enterprise-level open source VMM oriented to cloud data centers, implements a unified architecture to support Standard-VMs, containers and serverless (Micro-VM). StratoVirt has some competitive advantages, such as lightweight and low resource overhead, fast boot, hardware acceleration, and language-level security with Rust.
+
+Currently, StratoVirt in Kata supports Micro-VM machine type, mainly focus on FaaS cases, supporting device hotplug (virtio block), file-system sharing through virtio fs and so on.Kata Containers with StratoVirt now use virtio-mmio bus as driver, and doesn't support CPU/memory resize nor VFIO, thus doesn't support updating container resources after booted.
```
Comment (Apokleos) [inline docs/design/virtualization.md:119]: Please add a space before "Kata":  `...and so on.Kata Containers with StratoVirt...` -> `...virtio fs and so on. Kata Containers with StratoVirt...`

Comment (fadecoder) [reply docs/design/virtualization.md]: thanks，updated

**Review by Apokleos — APPROVED** [review]
Thx @fadecoder LGTM!

**Pull request merged**